packages/api/src/routes/contactRoute.js

```diff
@@ -16,8 +16,8 @@
 const express = require('express');
 const router = express.Router();
 const contactController = require('../controllers/contactController');
+const checkScope = require('../middleware/acl/checkScope');
 
-
-router.post('/', contactController.sendContactForm());
+router.post('/', checkScope(['add:contact']), contactController.sendContactForm());
```

Comment:
This is missing the "at my farm" validation, add hasFarmAccess to the route.

Reply:
@kikemarquez8 done